grai-server/app/telemetry/apps.py

```diff
@@ -0,0 +1,40 @@
+import os
+import uuid
+
+import posthog
+import sentry_sdk
+from django.apps import AppConfig
+from posthog.sentry.posthog_integration import PostHogIntegration
+from sentry_sdk import configure_scope
+from sentry_sdk.integrations.django import DjangoIntegration
+
+orgname = uuid.uuid4()
+PostHogIntegration.organization = orgname
+
+try:
+    # https://posthog.com/docs/integrate/server/python
+    # TODO: Possibly come back to this later
+    sentry_sdk.init(
+        dsn="https://examplePublicKey@o0.ingest.sentry.io/0",
```

Comment:
I presume this needs updating to a live key at some point?

Reply:
Good catch, thank you! Fix is up.